docs/wiki/Server-Admin.md

```diff
@@ -2,6 +2,15 @@
 
 ## Exchange Settings
 
+### Logo Image
+
+Server operators can optionally provide their preferred logo image when starting
+`dcrdex` using the `--logo` flag, e.g `dcrdex --logo path/to/logo.png`. Logo
+image should be preferably encoded as a `png`, `jpeg`, or `jpeg` file.
```

Comment:
Thanks for updating docs. We should word the instructions towards a configuration file, not CLI flags.